charts/metallb/templates/rbac.yaml

```diff
@@ -0,0 +1,165 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "metallb.fullname" . }}:controller
```

Comment:
I get a `helm lint` error here 
`"test-release-metallb:controller": invalid metadata name, must match regex ^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$ and the length must not be longer than 253`

Reply:
Interesting, ClusterRole names with `:` are definitely valid:
```
$ kubectl get clusterrole | grep system | head -1
system:discovery                                                       2021-01-01T17:10:36Z
```

Reply:
Ah, I left this note in `.circleci/config.yml`:
``` 20   # FIXME: this is pinned to v3.0.0 because subsequent versions of ct pull in new versions of helm, which are subject
 21   #        to this bug: https://github.com/helm/helm/issues/8596, specifically for rbac object names with ':' in them
 22   helm-lint:
 23     working_directory: /repo
 24     docker:
 25       - image: quay.io/helmpack/chart-testing:v3.0.0
```

Reply:
I am! This clear that up

Reply:
It appears they fixed the issue in helm/helm#8596 about deprecated API versions, but not the `:` issue.  I'll update the note in `.circleci/config.yml`

Reply:
Fixed, adjusted the comment about this in `.circleci/config.yml` to point to the new upstream issue for `:` in name.